<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JYUSign_in</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>账号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="userEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="signin">
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>密码：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>注销</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>IP:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ipEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="setButton">
          <property name="text">
           <string>配置</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>334</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>菜单(&amp;F)</string>
    </property>
    <addaction name="actiona"/>
    <addaction name="actiona_3"/>
    <addaction name="actiona_4"/>
    <addaction name="actiona_2"/>
   </widget>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actiona">
   <property name="text">
    <string>如何导入配置</string>
   </property>
  </action>
  <action name="actiona_2">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actiona_3">
   <property name="text">
    <string>设置开机自启</string>
   </property>
  </action>
  <action name="actiona_4">
   <property name="text">
    <string>取消开机自启</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
